week-09-js/var.js

```diff
@@ -0,0 +1,78 @@
+console.log("Hi!")
+
+
+getUsers();
+
+function renderListItem(userData) {
+  const li = document.createElement("li");
+  li.innerText = `${userData.lname} ${userData.fname} ${userData.Mphone} ${userData.street} ${userData.number} ${userData.City} ${userData.Country}`;
+  const ol = document.getElementById("fname","lname","Mphone","street","number","City","Country");
+  ol.appendChild(li);
+
```

Comment:
in order to see the the new created elements, you need to ether add the 'ol' element in the html doc, or created one with javascript and append it to the body